lib/services/service.js

```diff
@@ -24,30 +24,55 @@ const
   errorsManager = require('../config/error-codes/throw');
 
 /**
- * Services skeleton
+ * Services base class
```

Comment:
Service base class has been rewritten. Now it handles the init sequence with timeout